crates/oxc_ast/src/visit.rs

```diff
@@ -655,6 +655,7 @@ pub trait Visit<'a>: Sized {
     fn visit_elision(&mut self, span: Span) {
         let kind = AstKind::Elision(span);
         self.enter_node(kind);
+        /* cfg - currently elisions are not represented in the cfg */
```

Comment:
Better document this else where, this `Visit` trait is public facing.

Reply:
Done.